digdag-client/src/main/java/io/digdag/client/api/RestDirectDownloadHandle.java

```diff
@@ -1,20 +1,49 @@
 package io.digdag.client.api;
 
 import org.immutables.value.Value;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.JsonMappingException;
 
 @Value.Immutable
-@JsonSerialize(as = ImmutableRestDirectDownloadHandle.class)
-@JsonDeserialize(as = ImmutableRestDirectDownloadHandle.class)
 public interface RestDirectDownloadHandle
 {
-    String getType();
-
+    @JsonValue
     String getUrl();
 
-    static ImmutableRestDirectDownloadHandle.Builder builder()
+    @JsonCreator
+    @Deprecated
+    static RestDirectDownloadHandle of(JsonNode node)
```

Comment:
This is necessary because this structure:

```
{
    "type": "http",
    "url": "http://example.com"
}
```

is changed to:

```
"http://example.com"
```